src/provisioner.ts

```diff
@@ -20,32 +20,32 @@ import {
     MatrixRoom,
 } from "matrix-appservice-bridge";
 import * as Discord from "discord.js";
+import { DbRoomStore } from "./db/roomstore";
 
 const PERMISSION_REQUEST_TIMEOUT = 300000; // 5 minutes
 
 export class Provisioner {
 
-    private bridge: Bridge;
     private pendingRequests: Map<string, (approved: boolean) => void> = new Map(); // [channelId]: resolver fn
+    private roomStore: DbRoomStore;
 
-    public SetBridge(bridge: Bridge): void {
-        this.bridge = bridge;
+    public setStore(roomStore: DbRoomStore) {
```

Comment:
can't this go in the constructor? with #378 maybe?

Reply:
Probably with #378 :)